.github/workflow/release.yml

```diff
@@ -0,0 +1,105 @@
+﻿name: Publish
+
+on:
+  release:
+    types:
```

Comment:
You need [ released, prereleased ] here, that is intentional. Otherwise the workflow will not run when you publish a pre-release from a draft. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release